core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20171208_point_project_owners_to_policies.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="point-project-owners-to-policies-1">
+        <sql stripComments="true">
+            update WORKSPACE_ACCESS wa join WORKSPACE ws on ws.id = wa.workspace_id set wa.group_name = concat("owner@", ws.namespace, "@billing-project") where wa.access_level = "PROJECT_OWNER" and wa.group_name not like "%@%@%" and wa.group_name like "PROJECT_%-Owner"
```

Comment:
aren't the like and not like statements mutually exclusive? do you need both?

not that I really care

Reply:
there are also groups of the format `I_0415e505-a811-4208-b502-3cb17a462320-PROJECT_OWNER` used for actual intersection groups when there is an auth domain, so i need both `like`s in order to also exclude those from being updated